### ISSUE TITLE
RPi image change log

### ISSUE THREAD

**Issue by shoemakerlevy9** [UbiquityRobotics/ubiquity_main#166]
Is there a change log for your RPi images hosted anywhere. If not it would be a nice thing to have. We have been working with version 2018-1-13 for awhile which has some issues but haven't updated for fear of the unknown.

**Comment by rohbotics** [comment]
I agree that a changelog would be nice, but almost all of the changes on a new image are achieved by running `apt update; apt upgrade` on an older image, so I am unsure about how practical a change log would be.

In general we have not had any reports of problems caused by moving to newer images.
